app/src/androidTest/java/com/karumi/katasuperheroes/MainActivityTest.java

```diff
@@ -65,6 +85,70 @@
     onView(withText("¯\\_(ツ)_/¯")).check(matches(isDisplayed()));
   }
 
+  @Test public void showsEmptyCaseWhenThereAreSuperHeroes(){
+    givenThereAreOneSuperHeroe();
+    startActivity();
+    onView(withText("¯\\_(ツ)_/¯")).check(matches(not(isDisplayed())));
+  }
```

Comment:
It's a good practice to introduce a whitespace between the different test stages to improve the test readability. This test could be written as follows:

```java

givenThereAreOneSuperHeroe();

startActivity();

onView(withText("¯\\_(ツ)_/¯")).check(matches(not(isDisplayed())));
```